### ISSUE TITLE
fps option depricated? 

### ISSUE THREAD

**Issue by Hypercubed** [darlingjs/darlingjs#2]
The documention indicates that the fps can be specified when creating the world object.... but from what I can tell the rate is dictated by the window.requestAnimationFrame function.

**Comment by hyzhak** [comment]
@Hypercubed yes, you are right, but in near version 1.0 I will decoupled engine from `animation frame` #8 and it will be possible to use any tools for that
